.github/workflows/terraform-static-analysis.yml

```diff
@@ -0,0 +1,44 @@
+name: Terraform Static Code Analysis
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '**.tf'
+      - '.github/workflows/terraform-static-analysis.yml'
+
+jobs:
+  terraform-static-analysis:
+    name: Terraform Static Analysis
+    runs-on: ubuntu-latest
+    if: github.event_name != 'workflow_dispatch'
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2.3.4
+      with:
+        fetch-depth: 0
+    - name: Run Analysis
+      uses: ministryofjustice/github-actions/terraform-static-analysis@main
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        scan_type: changed
```

Comment:
There seems to be a bug in the github action which is causing the filenames being taken rather than the folder, probably changing this to "single" will fix it for now until the bug is fixed.